autoPyTorch/pipeline/components/training/trainer/AdversarialTrainer.py

```diff
@@ -109,11 +109,7 @@ def train_step(self, data: np.ndarray, targets: np.ndarray) -> Tuple[float, torc
         loss = loss_func(self.criterion, original_outputs, adversarial_outputs)
         loss.backward()
         self.optimizer.step()
-        if self.scheduler:
-            if 'ReduceLROnPlateau' in self.scheduler.__class__.__name__:
-                self.scheduler.step(loss)
-            else:
-                self.scheduler.step()
```

Comment:
because for now we only update the step in `train_epoch` of the `BaseTrainerComponent`. These lines were a bug as for `AdversarialTrainer` the scheduler is updated every batch as well as epoch